build-tools/create-pkg/create-pkg.csproj

```
@@ -1,33 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk" >
```

I actually think you should use `Microsoft.Build.NoTargets` here, because this isn't building C# code:

```suggestion
<Project Sdk="Microsoft.Build.NoTargets">
```